nitransforms/tests/data/v2v.lta

```diff
@@ -0,0 +1,30 @@
+# transform file v2v.lta
```

Comment:
How did you calculate this file?

Reply:
I used `mri_robust_register` on the the first volume of two bold runs:

```
$ mri_robust_register --mov run1.nii.gz --dst run2.nii.gz --lta r2r.lta
$ mri_robust_register --mov run1.nii.gz --dst run2.nii.gz --lta v2v.lta --vox2vox
```